.upstream-tests/test/CMakeLists.txt

```diff
@@ -37,11 +37,19 @@ else() # NOT LIBCUDACXX_TEST_WITH_NVRTC
   set(LIBCUDACXX_CUDA_COMPILER "${CMAKE_CUDA_COMPILER}")
 endif()
 
-set(LIBCUDACXX_TEST_COMPILER_FLAGS
-  "${LIBCUDACXX_TEST_COMPILER_FLAGS} \
-  ${LIBCUDACXX_FORCE_INCLUDE} \
-  -I${CMAKE_SOURCE_DIR}/include"
-  CACHE INTERNAL "Flags for libcxx testing." FORCE)
+if (NOT MSVC)
+  set(LIBCUDACXX_WARNING_LEVEL
+    "--compiler-options=-Wall \
+     --compiler-options=-Wextra")
+endif()
```

Comment:
I am not too happy with this. That said, we can reenable them once we have a workaround in place